cmake/thirdparty/FindCUDA.cmake

```diff
@@ -534,6 +552,13 @@ mark_as_advanced(
   CUDA_BUILD_EMULATION
   CUDA_VERBOSE_BUILD
   CUDA_SEPARABLE_COMPILATION
+  ####################
```

Comment:
These ``{BEGIN/END} BLT CHANGE`` comments should probably be removed before merging.

Reply:
we want those in there to make it very obvious what we changed w/o relying on git forensics

Reply:
Thanks @cyrush -- didn't notice that it was an edited version of cmake's ``FindCUDA``